strawberry/contrib/starlette/app/graphql_app.py

```diff
@@ -99,4 +99,5 @@ def __call__(self, scope: Scope) -> ASGIInstance:
             variable_values=variables,
             operation_name=operation_name,
             context_value=context,
+            root_value=self.schema.query_type
```

Comment:
I'm wondering if we should allow users to pass a custom root value 🤔 
what do you think?

Reply:
It looks like this will pass in the `Query` type as a root value for all requests, so using `self` in mutation or subscription resolvers would likely still throw an error.